Manito/Manito/Screens/CreateRoom/View/CreateRoomView.swift

```diff
@@ -0,0 +1,281 @@
+            $0.leading.trailing.equalToSuperview().inset(Size.leadingTrailingPadding)
+            $0.bottom.equalTo(self.nextButton.snp.top)
+        }
+
+        self.addSubview(self.roomDataCheckView)
+        self.roomDataCheckView.snp.makeConstraints {
+            $0.top.equalTo(self.titleLabel.snp.bottom).offset(66)
+            $0.leading.trailing.equalToSuperview().inset(Size.leadingTrailingPadding)
+            $0.bottom.equalTo(self.nextButton.snp.top)
+        }
+
+        self.bringSubviewToFront(self.nextButton)
+    }
+    
+    private func setupAction() {
+        let closeAction = UIAction { [weak self] _ in
+            self?.delegate?.didTapCloseButton()
+        }
+        self.closeButton.addAction(closeAction, for: .touchUpInside)
+        
+        let nextAction = UIAction { [weak self] _ in
+            self?.changeNextRoom()
+        }
+        self.nextButton.addAction(nextAction, for: .touchUpInside)
+        
+        let backAction = UIAction { [weak self] _ in
+            self?.changePreviousRoomIndex()
+        }
+        self.backButton.addAction(backAction, for: .touchUpInside)
+    }
+    
+    private func changeNextRoom() {
+        switch self.notiIndex {
+        case .inputTitle:
+            guard let text = self.roomTitleView.roomsNameTextField.text else { return }
+            self.title = text
+            self.setDataInCheckView(name: self.title)
+            self.changeNextRoomIndex()
+            self.changedInputView()
+            self.roomTitleView.roomsNameTextField.resignFirstResponder()
+        case .inputParticipants:
+            self.participants = Int(self.roomParticipantsView.personSlider.value)
+            self.setDataInCheckView(participants: self.participants)
+            self.changeNextRoomIndex()
+            self.changedInputView()
+        case .inputDate:
+            self.setDataInCheckView(date: "\(self.roomDateView.calendarView.getTempStartDate()) ~ \(self.roomDateView.calendarView.getTempEndDate())")
+            self.changeNextRoomIndex()
+            self.changedInputView()
+        case .checkRoom:
+            self.roomInfo = RoomDTO(title: self.title,
+                                    capacity: self.participants,
+                                    startDate: "20\(self.roomDateView.calendarView.getTempStartDate())",
+                                    endDate: "20\(self.roomDateView.calendarView.getTempEndDate())")
+            self.delegate?.pushChooseCharacterViewController(roomInfo: self.roomInfo)
+        }
+    }
+    
+    private func changePreviousRoomIndex() {
+        self.notiIndex = CreateRoomState.init(rawValue: self.notiIndex.rawValue - 1)!
+        self.changedInputView()
+    }
+    
+    private func changeNextRoomIndex() {
+        self.notiIndex = CreateRoomState.init(rawValue: self.notiIndex.rawValue + 1)!
+        self.changedInputView()
+    }
+    
+    private func changedInputView() {
```

Comment:
p5;

```suggestion
    private func changInputView() {
```

왠지 `changed`가 아니라 `change`가 되어야 할 것 같은 느낌... 이랄까요..??